src/main/java/com/powsybl/openloadflow/network/impl/AbstractLfBus.java

```diff
@@ -412,6 +412,13 @@ public double getLoadTargetP() {
                 .sum();
     }
 
+    @Override
+    public double getNonFictitiousLoadTargetP() {
```

Comment:
Are you sure there is no performance impact to compute this on the fly ?

Reply:
The laternative I see would be 
- to compute incrementally this value for *all* buses in each event modifying the loads.
    Seems complex and gain is not obvious
- to cache it at bus creation, ignoring deconnection events, actions etc..
     Might cause bugs

Here it is computed;
   at each restore, for each HVDC station
   at contingency propagation for each HVDC
   and you pay an iteration price if the bus has several loads.

This being said I'll run some mesures of this cost on a large network.

Reply:
It has ZERO performance impact the way it is used today.
The method was never called in a sensitivity analysis of a large network with 100 contingencies.
The reason is that restoreInitialTopology that will trigger the call for each HVDC link is called in very rare and targeted occasions - when some switches are moved. So I suggest to keep it simple stupid at this time.